lib/client/api.go

```diff
@@ -968,10 +968,10 @@ type TeleportClient struct {
 	// Useful to force flows that only typically happen on Teleport Enterprise.
 	dtAttemptLoginIgnorePing, dtAutoEnrollIgnorePing bool
 
-	// dtAuthnRunCeremony allows tests to override the default device
+	// dtAuthnCeremony allows tests to override the default device
 	// authentication function.
-	// Defaults to [dtauthn.RunCeremony].
-	dtAuthnRunCeremony dtAuthnRunCeremonyFunc
```

Comment:
You don't really need to change any of this, we can just assign `NewCeremony().Run` to it.